LayoutTests/http/tests/navigation/ping-attribute/secure-anchor-cross-origin-expected.txt

```diff
@@ -0,0 +1,6 @@
+Ping sent successfully
+CONTENT_TYPE: text/ping
```

Comment:
This test doesn't say pass or fail? How would one know if they regress it?

Reply:
The lack of HTTP_PING_FROM here is the important thing.  That's how all the tests in this directory are arranged.